sass/_universal.scss

```diff
@@ -0,0 +1,40 @@
+@import '_variables.scss';
+@import '_mixins.scss';
+/* UNIVERSAL */
+
+html {
+	box-sizing: border-box;
+
+	height: 100%;
+}
+
+*,
+*:before,
+*:after {
+	box-sizing: inherit;
+}
+
+html,
+body {
+	@include fontfamily(sans);
+}
+
+body {
+	min-width: $sm-mq;
+}
+
+// for bits like the close button on a modal, seems like this should be elsewhere...
+.close {
+	cursor: pointer;
+}
+
+.visible {
+	display: inline-block;
+}
+
+// get rid of that nasty space under an image when it's wrapped in an anchor
+.img {
+	a {
```

Comment:
Almost forgot, doesn't the nested ampersand need to precede selectors? Think I removed it because - 
```SCSS
.foo {
  & .bar {}
}
```
is allowed, and -
```SCSS
.foo {
  .bar & { }
}
```
is not.